### ISSUE TITLE
UnsatisfiableError: enum34 conflicts with Python 3.5

### ISSUE THREAD

**Issue by Benatti1991** [conda/conda#8054]
When I try to downgrade Python to 3.5 (PYTHON_VERSION = 3.5) following these steps:

```
   wget "http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"

    bash Miniconda3-latest-Linux-x86_64.sh -b

    export PATH=$HOME/miniconda3/bin:$PATH

    conda install python=$PYTHON_VERSION -y
```

I get stuck with this error:

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - conda[version='>=4.5.12'] -> enum34 -> python[version='>=2.7,<2.8.0a0']
  - python=3.5
```

I also noted that this error never appeared before today.

Thanks in advance,
Simone

**Comment by shulantianmayo** [comment]
I got exactly the same issue now when I tried to set up python3.5 with conda install python=3.5 (downloaded 3.7, but some libraries are only available in 3.5) in another mac laptop. Everything worked smoothly when I did this one month ago. Hopefully, this can be solved! thanks

**Comment by Benatti1991** [comment]
Everything works fine if I create a specific environment for python 3.5 instead of downgrading Conda.
